### DISCUSSION TITLE
Utiliser un export Enercoop

### DISCUSSION THREAD

**Discussion by TonyCrousty** [JC144/EDF_Simulateur_Prix#91] [Q&A] [answered]
Bonjour, voilà maintenant 2 semaine que j'essaye d'exporter ma conso horaires depuis mon espace client Enedis. Sans succès, ils m'annoncent un email à venir ... qui ne vient jamais. Le support Enedis vient de me répondre, je cite "il est normal que vous ne puissiez pas télécharger un export de l'historique complet de votre consommation horaire, car le serveur connaît actuellement une anomalie." :-/

Bref, tout cela pour dire que je voudrais bien pouvoir utiliser ce simulateur de prix à l'aide de l'export que j'ai pu obtenir auprès de mon fournisseur Enercoop. J'imagine que ne devrait pas être bien compliqué d'adapter son format à celui attendu pour un export EDF ou Enedis, mais je n'ai trouvé aucune spécifications. QUelqu'un saurait-il me renseigner sur les formats de CSV attendus ?

Ci-joint, pour info, l'export Enercoop.

Merci d'avance et bravo aux auteurs de ce super travail :)
[Exemple Export Conso Enercoop.csv](https://github.com/JC144/EDF_Simulateur_Prix/files/15121904/Exemple.Export.Conso.Enercoop.csv)

**Comment by TonyCrousty** [comment, marked as answer]
Trouvé en débuggant le code JS. Le format attendu pour un export Enedis (le nom du fichier doit contenir 'Enedis" pour entrer dans ce parseur) : 
2 colonnes CSV (séparateur ;) :
1. Date-heure au format YYYY-MM-DDTHH:mm (exemple : 2022-04-25T00:00)
2. Conso

**Reply by JC144** [reply]
Bonjour, désolé pour l'absence de réponse, j'ai un peu oublié la notif.

Super si vous avez pu trouver une solution, je regarderai à l'occasion pour intégrer les exports d'Enercoop.
Quelle syntaxe de noms de fichier ils utilisent ?

**Reply by TonyCrousty** [reply]
Je viens de refaire un téléchargement depuis mon espace client Enercoop. Ci-joint le fichier non modifié, dont j'ai juste supprimé la plupart des lignes.
[19976700381123_consommation_horaire_2021-05-26_2024-05-25.csv](https://github.com/JC144/EDF_Simulateur_Prix/files/15448937/19976700381123_consommation_horaire_2021-05-26_2024-05-25.csv)
